contrib/meson.build

```diff
@@ -5,13 +5,13 @@ glib_dep = dependency('glib-2.0')
 m_dep = cc.find_library('m', required: false)
 capng_dep = dependency('libcap-ng', required: get_option('capng'))
 ncurses_dep = dependency('curses', required: get_option('ui'))
-numa_dep = dependency('numa')
+numa_dep = dependency('numa', required: get_option('numa'))
 systemd_dep = dependency('libsystemd', required: get_option('systemd'))
 
 cdata = configuration_data()
 cdata.set('HAVE_GETOPT_LONG', cc.has_function('getopt_long'))
 cdata.set('HAVE_IRQBALANCEUI', ncurses_dep.found())
-cdata.set('HAVE_NUMA_H', cc.has_header('numa.h'))
+cdata.set('HAVE_NUMA_H', cc.has_header('numa.h') and numa_dep.found())
 cdata.set('HAVE_LIBCAP_NG', capng_dep.found())
 cdata.set('HAVE_LIBSYSTEMD', systemd_dep.found())
```

Comment:
I understand you didn't write it this way, but autotools default enables systemd support if the library is located, and offers an option to disable it.  If we're starting to consider a switch to meson, shouldn't there be an option to disable it as well?

Reply:
The build option for systemd integration is exposed via the entry in `meson_options.txt`.
As mentioned above, one would either specify the preference via `meson -Dsystemd=enabled`, or by using `--auto-features enabled` (to enable all features).

Currently, the workflow is structured in such a way, that all wanted features need to be toggled on (unless we would add a `value: enabled` to the entries in `meson_options.txt`). This is because the `get_option()` call in the [dependency()](https://mesonbuild.com/Reference-manual_functions.html#dependency) call further up in this file defines whether the systemd dependency is searched for.

Reply:
I have enabled the systemd feature by default in an additional commit.